xla/service/gpu/BUILD

```diff
@@ -82,6 +81,7 @@ tf_proto_library(
 xla_cc_test(
     name = "backend_configs_test",
     srcs = ["backend_configs_test.cc"],
+    tags = tf_gpu_tests_tags(),
```

Comment:
If this test had no custom tags before, does it need them now?

Reply:
@cheshire  Since some of the unit tests do no have custom tags, they were not selected to run as part of CI as they were filtered out. Motivation for this PR is to identify such unit tests and run them as part of CI.